src/main/javascript/view/OperationView.js

```diff
@@ -19,6 +19,15 @@ SwaggerUi.Views.OperationView = Backbone.View.extend({
     this.parentId = this.model.parentId;
     this.nickname = this.model.nickname;
     this.model.encodedParentId = encodeURIComponent(this.parentId);
+
+    // Fix for duplicate ? in queries where the path already
+    // contains a query parameter
+    this.model.rawPath = this.model.path;
+    var tokens = this.model.rawPath.split('?');
+    if (tokens.length > 1) {
+      this.model.path = tokens[0];
+    }
```

Comment:
@bhomnick so the problem i'm seeing with this (and it may just be my ignorance of the feature, or your current progress) is now we're stripping out the category param, which yields no tutoring results. I get this response:

```json
{
  "message": "Please provide a valid category.",
  "errors": {},
  "success": false
}
```

with this request url:

```
http://local.noodle.org/api/v1/search/?per_page=20&page=1
```

I imagine that's not intended?